### DISCUSSION TITLE
Enabling npm Package Provenance

### DISCUSSION THREAD

**Discussion by gamemaker1** [nfriedly/express-rate-limit#406] [General]
See https://github.blog/2023-04-19-introducing-npm-package-provenance/.

Seems easy enough to enable, we just need to pass a `--provenance` flag to `npm` the `publish` step of the CI.

Also, it gives us a fancy green tick next to the version on the npm page ;)

**Comment by nfriedly** [comment]
Yeah, that seems like a good idea

**Comment by gamemaker1** [comment]
Made a pr for the same on the `rate-limit-postgresql` repo, and released patch versions with the change for the rest.
